src/hooks/post-detail/useComments.ts

```
@@ -0,0 +1,138 @@
+
+import { useEffect, useRef, useState } from "react";
+import type { PostComment, PostDetailUnion } from "@/types/postDetail";
+import { PostDetailService } from "@/services/postDetail";
+import { useAlertStore } from "@/store/alert";
+import { useModalStore } from "@/store/modal";
+import { useApollo } from "@/lib/apolloClient";
+
+export function useComments(postInfo: PostDetailUnion | null) {
+  const client = useApollo();
+  const { success, error } = useAlertStore();
+  const { open: openModal, close: closeModal, isOpen } = useModalStore();
```

ESLint

[error] 'isOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)